index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+        <!--TOOLBAR SECTION-->
+        <nav class="toolbar">
+            <div class="my-logo primary-color hover-underline">My Logo</div>
```

Comment:
Kindly wrap your logo in an `<a>` tag🙏. Typically, the logo links to the homepage of a website. This action helps improve user experience